third-party-licenses.csv

```diff
@@ -1,12 +1,4 @@
 "module name","licenses","repository","licenseUrl","parents"
-"@docusaurus/core@2.0.0-beta.15","MIT","https://github.com/facebook/docusaurus","https://github.com/facebook/docusaurus/raw/master/LICENSE","website"
-"@docusaurus/preset-classic@2.0.0-beta.15","MIT","https://github.com/facebook/docusaurus","https://github.com/facebook/docusaurus/raw/master/LICENSE","website"
-"@mdx-js/react@1.6.22","MIT","https://github.com/mdx-js/mdx","https://github.com/mdx-js/mdx/raw/master/license","website"
 "@types/lodash@4.14.168","MIT","https://github.com/DefinitelyTyped/DefinitelyTyped","https://github.com/DefinitelyTyped/DefinitelyTyped/raw/master/LICENSE","cursorless"
-"clsx@1.1.1","MIT","https://github.com/lukeed/clsx","https://github.com/lukeed/clsx/raw/master/license","website"
 "immutability-helper@3.1.1","MIT","https://github.com/kolodny/immutability-helper","https://github.com/kolodny/immutability-helper/raw/master/LICENSE","cursorless"
-"mdast-util-find-and-replace@2.1.0","MIT","https://github.com/syntax-tree/mdast-util-find-and-replace","https://github.com/syntax-tree/mdast-util-find-and-replace/raw/master/license","website"
-"prism-react-renderer@1.2.1","MIT","https://github.com/FormidableLabs/prism-react-renderer","https://github.com/FormidableLabs/prism-react-renderer/raw/master/LICENSE","website"
-"react-dom@17.0.2","MIT","https://github.com/facebook/react","https://github.com/facebook/react/raw/master/LICENSE","website"
-"react@17.0.2","MIT","https://github.com/facebook/react","https://github.com/facebook/react/raw/master/LICENSE","website"
-"unist-util-visit@4.1.0","MIT","https://github.com/syntax-tree/unist-util-visit","https://github.com/syntax-tree/unist-util-visit/raw/master/license","website"
\ No newline at end of file
+"lodash@4.17.21","MIT","https://github.com/lodash/lodash","https://github.com/lodash/lodash/raw/master/LICENSE","cursorless"
```

Comment:
I think this change is an automated one by the build system or something. I've seen it pop up during my own changes. Not sure if it should actually be included. @pokey?